### PR TITLE
add more mouse buttons

### PR DESCRIPTION
**Pull request by harbdog**
- updating to ebitengine v2.5 which introduced [more mouse button inputs](https://github.com/hajimehoshi/ebiten/commit/9e9cec249a08bbf9d21cf132fb21fdbea5f8c574)
- added input mouse keys for buttons 3&4, corresponding to the back/forward browser buttons

### CODE REVIEW

**Comment by tinne26** [comment]
Probably `ColorScale.ScaleWithColor()` should be preferred. Also, minor, but why `&c` instead of just `c`? Is there any performance consideration I'm not aware of? (seems kinda superfluous otherwise).

**Comment by quasilyte** [comment]
I can merge a Scale->ScaleWithColor fix in a separate PR.

**Comment by quasilyte** [comment]
I wanted to stay on a pre-2.5 version in this library due to a performance regression that happened after v2.5.0-alpha.12.0.20230228174701-7c0fbce0cfd8, but I guess I'll need to update at some point anyway.

**Comment by harbdog** [comment]
> Probably `ColorScale.ScaleWithColor()` should be preferred. Also, minor, but why `&c` instead of just `c`? Is there any performance consideration I'm not aware of? (seems kinda superfluous otherwise).

Good call, sorry I had only looked at what I had done in one of my projects to handle the deprecation and realized I only did it like that because that code didn't have a color object being passed around.

File: go.mod [hunk 1]
```diff
@@ -3,17 +3,17 @@ module github.com/quasilyte/ebitengine-input
...
 
 require (
-	github.com/ebitengine/purego v0.0.0-20220905075623-aeed57cda744 // indirect
-	github.com/go-gl/glfw/v3.3/glfw v0.0.0-20220806181222-55e207c401ad // indirect
-	github.com/hajimehoshi/file2byteslice v0.0.0-20210813153925-5340248a8f41 // indirect
-	github.com/jezek/xgb v1.0.1 // indirect
+	github.com/ebitengine/purego v0.4.0 // indirect
+	github.com/go-gl/glfw/v3.3/glfw v0.0.0-20221017161538-93cebf72946b // indirect
+	github.com/jezek/xgb v1.1.0 // indirect
 	golang.org/x/exp v0.0.0-20190731235908-ec7cb31e5a56 // indirect
```
Comment (hajimehoshi) [inline go.mod:14]: I recommend removing this line and go-mod-tidying again.

**Pull request merged**